RELEASING.md

```diff
@@ -5,16 +5,9 @@ Client](https://github.com/googleapis/google-cloud-ruby) project.
 
 ## Releasing individual gems and meta-packages
 
-The Google Cloud Ruby Client project uses [semantic
-versioning](http://semver.org). Replace the `<version>` placeholder in the
-examples below with the appropriate number, e.g. `0.1.0`. Replace the `<gem>`
-placeholder with the appropriate full name of the gem, e.g.
-`google-cloud-datastore`.
-
-After all [pull
+After all relevant [pull
 requests](https://github.com/googleapis/google-cloud-ruby/pulls) for a
-release have been merged and all Kokoro and [Circle CI
-builds](https://circleci.com/gh/googleapis/google-cloud-ruby) are
+release have been merged and all Kokoro builds are
```

Comment:
I think it's time to add a line telling people how to check the Kokoro builds. :(

Reply:
Maybe refer to the final step, or copy that content and paste it here.